src/client/containers/CreateBoardPage/CreateBoardPage.container.js

```diff
@@ -1,8 +1,11 @@
 import React, { useState } from 'react';
 import AddMembers from '../AddMembersPage/AddMembersPage.container';
+import { useUser } from '../../firebase/UserContext';
 
 export default function CreateBoard() {
-  const [members, setMembers] = useState([]);
+  const { user } = useUser();
+  const userId = user[0].id;
+  const [members, setMembers] = useState([userId]);
```

Comment:
Ok, I see you add the userId to the members before you pass the members list.

Reply:
It's a bit imperative. Firstly I would call it membersWithOwner
Also, do try and create subcomponents and avoid weaving ternary statements inbetween the jsx. In AddMembers you should have a MembersList component.